src/Widgets/HeadlineWidget/HeadlineWidget.scss

```diff
@@ -0,0 +1,9 @@
+.headline-widget {
+  // Fix navigation overlapping anchor issue
+  // See .navbar "min-height" value
+  // Source: https://stackoverflow.com/a/38106970/881759
+  &-anchor {
```

Comment:
In other widget style files we use the double-dash separator, for example https://github.com/Scrivito/scrivito_example_app_js/blob/master/src/Widgets/PriceWidget/PriceWidget.scss#L5

* [ ] Please check